CHANGELOG.md

```diff
@@ -43,6 +43,19 @@ Disabling links that are styled to look like buttons will not be supported by fu
 
 This was added in [pull request #3326: Deprecate `govuk-button--disabled` class](https://github.com/alphagov/govuk-frontend/pull/3326).
 
+#### Stop using the `govuk-if-ie8` and `govuk-not-ie8` mixins and the `$govuk-is-ie8` and `$govuk-ie8-breakpoint` settings
```

Comment:
Alternative headings:
Stop using the mixins `govuk-if-ie8` and `govuk-not-ie8` and the breakpoint settings `$govuk-is-ie8` and `$govuk-ie8-breakpoint`
or
Stop using the mixins and breakpoint settings for generating IE8 stylesheets
or 
Stop using the deprecated IE8 mixins and settings.

It depends how much detail you want in the header.

Reply:
@36degrees I've made some suggestions, but the rest looks fine to me.